phpunit.xml

```diff
@@ -16,6 +16,8 @@
     <testsuites>
         <testsuite name="tests">
             <directory>tests</directory>
+            <exclude>tests/smbo</exclude>
```

Comment:
yeah, I've added a few test-models, it won't have any data, but it should contain some interesting test-cases here. It should be OK as no sensitive data is shared.